src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -225,7 +225,7 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
      * Return true if ICE is connected.
      */
     fun isIceConnected(): Boolean {
-        val result = driver.executeScript("return APP.conference.getConnectionState()")
+        val result: Any? = driver.executeScript("return APP.conference.getConnectionState()")
```

Comment:
Might be worth following the pattern used by the other `executeScript` calls, where we match on the type of the result.  This also allows you to log what was returned if it's not what was expected (which can happen if `APP.conference` isn't defined or something, or if the name of the `getConnectionState` function changes)